configure.ac

```diff
@@ -28,7 +28,7 @@ AC_CONFIG_MACRO_DIR([m4])
 
 AM_INIT_AUTOMAKE([-Wall -Wno-portability subdir-objects foreign dist-bzip2 no-dist-gzip])
 
-AC_PROG_CC_C89
+AC_PROG_CC
```

Comment:
Yes, but as described in #821, `AC_PROG_CC` calls `AC_PROG_CC_C89`, the inclusion in the other direction is not documented